### PR TITLE
WIP: Add build matrix entry for native Debian versions

### LINKED ISSUE #38: texttable-0.8.1
Currently the minimum version of texttable is >=0.8.2 , but I cant find a bug in 0.8.1.

See two commits in https://github.com/jayvdb/pyexcel/commits/texttable-0.8.1 , and successful build at https://travis-ci.org/jayvdb/pyexcel/builds/126812754 .

If there are serious bugs in 0.8.1, we should add a test that fails on 0.8.1.

Debian sid and ubuntu trusty-yakkety all provide 0.8.1 as OS packages, so it would be good if we could use 0.8.1, or request that debian/ubuntu bump their version if there is a serious bug.

https://packages.debian.org/search?suite=sid&searchon=names&keywords=python-texttable
http://packages.ubuntu.com/trusty/python-texttable
http://packages.ubuntu.com/yakkety/python-texttable

### PR DESCRIPTION
**Pull request by jayvdb**
Underlying work to fix #38

### CODE REVIEW

File: rnd_requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 https://github.com/pyexcel/pyexcel-io/archive/master.zip
-https://github.com/pyexcel/pyexcel-xls/archive/master.zip
+https://github.com/jayvdb/pyexcel-xls/archive/use-sid.zip
```
Comment (jayvdb) [inline rnd_requirements.txt]: this needs to be reverted once the change to pyexcel-xls has been merged

**Comment by jayvdb** [comment]
bah ; see comment -- this needs to be amended before being merged

**Comment by chfw** [comment]
Saw it will update the reference later.

**Comment by jayvdb** [comment]
ok; #38 needs reopening as that isnt done yet

**Pull request merged**